tests/tftpgen_test.py

```diff
@@ -7,6 +7,26 @@
 from cobbler.items.distro import Distro
 from tests.conftest import does_not_raise
 
+# Tests copying the bootloaders from the bootloaders_dir (setting specified in /etc/cobbler/settings.yaml) to the tftpboot directory.
+def test_copy_bootloaders():
+    # Instantiate TFTPGen class with collection_mgr parameter
+    test_api = CobblerAPI()
+    test_collection_mgr = CollectionManager(test_api)
+    generator = tftpgen.TFTPGen(test_collection_mgr)
+
+    # Arrange
+    ## Dummy/empty bootloader files are staged in 'test_data'. See below "cp" command.
+    os.system("cp -f /code/tests/test_data/dummy_bootloaders/* /var/lib/cobbler/loaders/")
```

Comment:
The pythonic way for doing this would be something like this: https://stackoverflow.com/a/18372078/4730773

```suggestion
    import glob
    import shutil
    dest_dir = "/var/lib/cobbler/loaders/"
    for file in glob.glob(r'/code/tests/test_data/dummy_bootloaders/*'):
        shutil.copy(file, dest_dir)
```

P.S.: Please sort the imports to the top. This is just a poc how you code could look like.